lib/OpenQA/UserAgent.pm

```diff
@@ -30,8 +30,9 @@ sub new {
             for my $i (qw(key secret)) {
                 my $attr = "api$i";
                 next if $self->$attr;
-                my $val = $cfg->val($args{api}, $i);
-                next unless $val;
+                # Fetch all the values in the file and keep the last one
+                my @values = $cfg->val($args{api}, $i);
+                next unless my $val = $values[-1];
```

Comment:
I feel like this could lead to confusing results? How about just failing with a clear error message? Something like:

```suggestion
                my $val = $cfg->val($args{api}, $i);
                die "$i was specified multiple times for $api" if $val =~ /[\n]/;
                next unless $val;
```

Reply:
That will break the bootstrap script when it's called multiple times.

A more correct solution would be overwriting the section but that requires more structured code than the bare `echo >> file` in the bootstrap script.

And then validating that sections are not repeated will not be a breaking change.